src/lib/Libwin/util.c

```diff
@@ -251,6 +252,8 @@ fcntl(int fd, int cmd, long arg)
 char *
 realpath(const char *path, char *resolved_path)
 {
+	if (resolved_path == NULL)
+		resolved_path = (char *)malloc(PATH_MAX);
```

Comment:
No need to cast the return value of malloc().  It returns a void * which is a generic pointer.

Reply:
OK will remove.